project-reports/2024/2024-annual-Hyperledger-Indy.md

```diff
@@ -0,0 +1,84 @@
+
+The Hyperledger Indy ecosystem community came together at a series of Hyperledger Indy Ecosystem Summits (organized by the Sovrin Foundation), bringing together network operators, node operators, project maintainers, network users, vendors to discuss the state of Indy and proposals for moving the project forward. The summits were well-attended (up to 90 participants), a better understanding of the community of interest was gained, and next steps were proposed, with some (mostly notably, the Indy-Besu initiative) acted upon.
+
+## Maintainer Diversity
+
+There are 35 individuals on 22 Indy GitHub Teams representing at least 16 organizations. We continue to have participation from (more or less) the same organizations as in 2022 — which is both good (still participating) and bad (few newcomers).
+
+From the last Quarterly Report: _Still needed -- a cleanup of the teams are needed as some of the individuals are no longer active in the community. No progress was made on that cleanup this quarter. We have the team lists, but have not queried those on the list to see if they are still interested in being maintainers._
+
+Typical Indy [Maintainers File](https://github.com/hyperledger/indy-plenum/blob/main/MAINTAINERS.md).
+
+## Project Adoption
+
+Insights from the Hyperledger Indy Ecosystem Summit showed that project adoption is “stable”. It seems those that have adopted Indy continue to stay with it and while there is evidence of new private deployments, we’re not aware of new public ones. There remains little competition for what Indy is today — an easy to deploy, stable, token-less, public, permissioned blockchain that is purpose-built for privacy-preserving verifiable credentials. The best alternative to Indy, and one being investigated as a potential Indy replacement, is Besu, with features that align with the existing Indy concepts.
+
+It should be noted that we continue to see a steady stream of people deploying Indy through questions asked on Discord and in GitHub issues, but we do not have visibility into why they are deploying Indy and if they are continuing to use it.
+
+## Goals
+
+### Performance Against Prior Goals
+
+As this is the first Hyperledger Indy Annual Report, there is not a 2022 Annual Report to look back on. However, in March 2023, the Indy Contributors group developed this [Indy Roadmap](https://hackmd.io/GeRP00i0Sj-7z4zXn2MB5g?view), an (non-timelined) wish list of technical capabilities we’d like to see in Indy. That roadmap was part of the agenda of the Hyperledger Indy Ecosystem summit, where it was highlighted that in addition to the technical capabilities, a focus was needed on marketing and on influencing regulatory guidance on decentralized identity infrastructure.
```

Comment:
Is this roadmap URL referenced in any of the Indy repos? I couldn't find it, at least in the READMEs, though I may have missed something.

Reply:
It is referenced and discussed at the Indy Contributor meetings. I don’t believe it is in any repos — there is not really an umbrella Indy repo — Indy has a set of repos.

From the Wiki: https://wiki.hyperledger.org/display/indy/Indy+Contributors+Meeting